packages/spritesheet/src/Spritesheet.ts

```diff
@@ -388,7 +428,7 @@ export class Spritesheet<S extends ISpritesheetData = ISpritesheetData>
                 );
 
                 // lets also add the frame to pixi's global cache for 'from' and 'fromLoader' functions
-                Texture.addToCache(this.textures[i], i.toString());
+                Texture.addToCache(this.textures[i], this.cachePrefix + i.toString());
```

Comment:
This would certainly resolve my use case of needing to namespace same-named textures in multiple loaded spritesheets, so looks good to me!